lib/dark_worlds_server/matchmaking/matching_coordinator.ex

```diff
@@ -98,4 +102,11 @@ defmodule DarkWorldsServer.Matchmaking.MatchingCoordinator do
     Process.send_after(client_pid, {:game_started, game_pid, game_config}, 1_000)
     consume_and_notify_players(rest_players, game_pid, game_config, count - 1)
   end
+
+  defp notify_players_amount(players, capacity) do
+    players
+    |> Enum.each(fn {_, client_pid} ->
+      Process.send(client_pid, {:notify_players_amount, Enum.count(players), capacity}, [])
```

Comment:
`Enum.count(players)` should be done outside the loop

Reply:
Woops! done